src/services/storage/models/offer.model.ts

```diff
@@ -52,3 +54,21 @@ export default class Offer extends Model {
     return this.totalCapacity.minus(this.utilizedCapacity)
   }
 }
+
+export async function getStakesAggregateQuery (sequelize: Sequelize, currency: 'usd' | 'eur' | 'btc' = 'usd') {
```

Comment:
Can you add JSDoc explaining what this does?

Reply:
Will do